From 62fa8c0f5adc8498a825ef2839d1bb216ea94783 Mon Sep 17 00:00:00 2001
From: Limeng <Meng.Li@windriver.com>
Date: Sat, 29 Sep 2018 16:49:10 +0800
Subject: [PATCH] u-boot: arm: libfdt: Remove leading underscores

Remove leading underscores leading underscores of _LIBFDT_ENV_H
in u-boot code. This fix refer to u-boot upstream db405d1980e0
("scripts/dtc: Update to upstream version v1.4.6-9-gaadd0b65c987").

Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 include/libfdt.h     | 6 +++---
 include/libfdt_env.h | 6 +++---
 2 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/include/libfdt.h b/include/libfdt.h
index a1ef1e1..6ba250c 100644
--- a/include/libfdt.h
+++ b/include/libfdt.h
@@ -1,5 +1,5 @@
-#ifndef _LIBFDT_H
-#define _LIBFDT_H
+#ifndef LIBFDT_H
+#define LIBFDT_H
 /*
  * libfdt - Flat Device Tree manipulation
  * Copyright (C) 2006 David Gibson, IBM Corporation.
@@ -1638,4 +1638,4 @@ int fdt_find_regions(const void *fdt, char * const inc[], int inc_count,
 		     struct fdt_region region[], int max_regions,
 		     char *path, int path_len, int add_string_tab);
 
-#endif /* _LIBFDT_H */
+#endif /* LIBFDT_H */
diff --git a/include/libfdt_env.h b/include/libfdt_env.h
index 273b5d3..992fbce 100644
--- a/include/libfdt_env.h
+++ b/include/libfdt_env.h
@@ -6,8 +6,8 @@
  * SPDX-License-Identifier:	LGPL-2.1+
  */
 
-#ifndef _LIBFDT_ENV_H
-#define _LIBFDT_ENV_H
+#ifndef LIBFDT_ENV_H
+#define LIBFDT_ENV_H
 
 #include "compiler.h"
 #include "linux/types.h"
@@ -26,4 +26,4 @@ typedef __be64 fdt64_t;
 /* adding a ramdisk needs 0x44 bytes in version 2008.10 */
 #define FDT_RAMDISK_OVERHEAD	0x80
 
-#endif /* _LIBFDT_ENV_H */
+#endif /* LIBFDT_ENV_H */
-- 
2.7.4

